fakeserver/server.go

```diff
@@ -24,20 +22,15 @@ type server struct {
 	router *mux.Router
 }
 
-// Start the server
-func Start(listenOn string) {
-	var wait time.Duration
-	flag.DurationVar(&wait, "graceful-timeout", time.Second*15, "the duration for which the server gracefully wait for existing connections to finish - e.g. 15s or 1m")
-	flag.Parse()
```

Comment:
I removed this flag stuff since it was unused.

Reply:
Thanks for catching.